examples/demo3/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+echo "mode: advantedge" >app_instance.yaml
+echo "sandbox:" >>app_instance.yaml
+echo "mecplatform: ${MEEP_MEP_NAME}" >>app_instance.yaml
+echo "appid:" ${MEEP_APP_ID} >>app_instance.yaml
+echo "localurl: ${MEEP_POD_NAME}" >>app_instance.yaml
+echo "port:" >>app_instance.yaml
```

Comment:
Is it necessary to include configuration parameters that are not used?

```suggestion
echo "mode: advantedge" >app_instance.yaml
echo "mecplatform: ${MEEP_MEP_NAME}" >>app_instance.yaml
echo "appid:" ${MEEP_APP_ID} >>app_instance.yaml
echo "localurl: ${MEEP_POD_NAME}" >>app_instance.yaml
```